examples/__tests__/test-utils.ts

```diff
@@ -1,7 +1,27 @@
 import { basename, join } from 'path';
-import { testDeployment } from '../../test/lib/deployment/test-deployment.js';
+import { lstatSync, readdirSync } from 'fs';
 
 export async function deployExample(filename: string) {
+  const { testDeployment } = require('../../test/lib/deployment/test-deployment.js');
```

Comment:
Non-blocking: it might be a good idea to drop a comment as to why this is being required inside `deployExample()` instead of a top-level `import`.

Reply:
Seems like jest had a problem with it, perhaps because of the package boundary. Or maybe it had a typo, not sure. I can check again when I’m not AFK